website/src/pages/[organism]/api/sequences.ts

```diff
@@ -226,7 +226,7 @@ function streamFasta(
     const sequenceName = searchParams.segment ?? 'main';
     const ndjsonLineSchema = z.object({
         [ACCESSION_VERSION_FIELD]: z.string(),
-        [sequenceName]: z.string(),
+        [sequenceName]: z.string().nullable(),
```

Comment:
just curious was there an error here or before and this was always nullable? If not why can it now be nullable?